gradle.properties

```diff
@@ -19,7 +19,7 @@ commonsioVersion=2.5
 commonsLang3Version=3.6
 findbugsVersion=3.0.1
 gradleDockerPlugin=3.1.0
-grpcVersion=1.6.1
+grpcVersion=1.8.0
```

Comment:
Why are these changes needed? Every time we increment changes we risk introducing regressions.

Reply:
grpc interceptors and SSL has been changing quiet a bit. The latest one is with 1.8.0 (and is supposed to be stable).